### PR TITLE
XW-2232 | Add partner slot to Global Navigation and Site Head

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Links

* http://wikia-inc.atlassian.net/browse/XW-2232

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/templates/components/site-head-fandom-bar.hbs [hunk 1]
```diff
@@ -2,4 +2,10 @@
 	<a href="{{wikiaHomepage}}">
 		{{svg 'wds-company-logo-powered-by-wikia' role='img' class='fandom-bar-logo'}}
 	</a>
+	<a class="site-head-fandom-bar__partner-logo" href="http://www.entertainweb.de/">
+		<img src="https://services.wikia.com/static-assets/image/24fc42b9-5429-49f3-8515-130626fb4751"
+			 height="20px"
+			 width="67px"
```
Comment (rogatty) [inline front/main/app/templates/components/site-head-fandom-bar.hbs]: Why it's not in the CSS?

File: front/main/app/styles/component/_site-head-fandom-bar.scss [hunk 2]
```diff
@@ -42,6 +42,13 @@ $home-of-fandom-letter-spacing: 0.5px;
 		height: 20px;
 		width: 130px;
 	}
+
+	&__partner-logo {
+		align-items: center;
+		display: flex;
```
Comment (rogatty) [inline front/main/app/styles/component/_site-head-fandom-bar.scss]: We should display it only on German wikis, not always.

Comment (tomasznapieralski) [inline front/main/app/styles/component/_site-head-fandom-bar.scss]: i know, it was not finished yet :)

File: front/main/app/templates/components/site-head-fandom-bar.hbs [hunk 3]
```diff
@@ -2,4 +2,12 @@
 	<a href="{{wikiaHomepage}}">
 		{{svg 'wds-company-logo-powered-by-wikia' role='img' class='fandom-bar-logo'}}
 	</a>
+	{{#if displayPartnerLogo}}
+		<a class="site-head-fandom-bar__partner-logo" href="http://www.entertainweb.de/">
+			<img src="https://services.wikia.com/static-assets/image/24fc42b9-5429-49f3-8515-130626fb4751"
+				 height="20px"
```
Comment (rogatty) [inline front/main/app/templates/components/site-head-fandom-bar.hbs]: Why it's not in the CSS?

File: front/main/app/templates/components/design-system/global-navigation.hbs [hunk 4]
```diff
@@ -49,4 +49,11 @@
 			</a>
 		</div>
 	</div>
+	{{# if model.partner_slot }}
```
Comment (rogatty) [inline front/main/app/templates/components/design-system/global-navigation.hbs]: It's not Mediawiki, mister!

File: front/main/bower.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Autolinker.js": "0.22.0",
     "cropper": "0.11.0",
-    "design-system": "Wikia/design-system#0.14.4",
+    "design-system": "Wikia/design-system#XW-2232",
```
Comment (rogatty) [inline front/main/bower.json]: Remember, remember.

File: front/main/ember-cli-build.js [hunk 6]
```diff
@@ -144,19 +144,15 @@ module.exports = function (defaults) {
 		numeralAssets = new Funnel(app.bowerDirectory + '/numeral/languages', {
 			destDir: 'assets/vendor/numeral'
 		}),
-		designSystemIconsAssets = new Funnel(app.bowerDirectory + '/design-system/dist/svg/sprite-icons.svg', {
+		designSystemAssets = new Funnel(app.bowerDirectory + '/design-system/dist/svg/sprite.svg', {
 			destDir: 'assets/design-system-icons.svg'
```
Comment (rogatty) [inline front/main/ember-cli-build.js]: Please update this too.

**Pull request merged**